homeassistant/components/habitica/manifest.json

```diff
@@ -1,7 +1,8 @@
 {
-  "domain": "habitica",
-  "name": "Habitica",
-  "documentation": "https://www.home-assistant.io/integrations/habitica",
-  "requirements": ["habitipy==0.2.0"],
-  "codeowners": []
+    "domain": "habitica",
+    "name": "Habitica",
+    "config_flow": true,
+    "documentation": "https://www.home-assistant.io/integrations/habitica",
+    "requirements": ["habitipy==0.2.0"],
+    "codeowners": ["@ASMfreaK", "@leikoilja", "@MartinHjelmare"]
```

Comment:
Please don't add me as code owner.

Reply:
Ou, okay, as you wish :) You just made so many adjustments and suggestions that I felt that giving you credits is a least i should do, haha.
No worries, will remove

Reply:
Ou, sure, as you wish :)
You were just so helpful and made so many valid changes and contributions that i thought adding you as a contributor is a least i can do.